### PR TITLE
ASM-3682 All exports should now reset idrac if it fails

### PR DESCRIPTION
**Pull request by asaenz**

### CODE REVIEW

**Comment by gavin-scott** [comment]
¿Dónde está mi jira?

**Comment by asaenz** [comment]
@gavin-scott It's in the title

File: lib/puppet/provider/idrac.rb [hunk 1]
```diff
@@ -226,23 +220,33 @@ def setup_idrac
...
-          transport[:password],
-          resource,
-          '/var/nfs',
-          postfix
-      )
-      obj.exporttemplatexml
+    begin
+      execute_export_config(postfix)
+    rescue
+      Puppet.debug 'Export template failed.'
```
Comment (gavin-scott) [inline lib/puppet/provider/idrac.rb]: Can you add the exception to the debug message?

**Comment by asaenz** [comment]
@gavin-scott Added the exception to the debug message as requested

**Pull request merged**